### PR TITLE
connect with dial

### PR DESCRIPTION
**Pull request by morefreeze**
PTAL

### CODE REVIEW

**Comment by siddontang** [comment]
LGTM

File: client.go [hunk 1]
```diff
@@ -30,42 +30,34 @@ import (
...
 }
 
 const (
 	initConnection int           = 5
 	maxConnecion   int           = 100
-	readTimeout    time.Duration = 5 // seconds
-	writeTimeout   time.Duration = 5 // seconds
+	readTimeout    time.Duration = 5 * time.Second // seconds
+	writeTimeout   time.Duration = 5 * time.Second // seconds
+	clientTimeout  time.Duration = 5 * time.Second // seconds
```
Comment (ngaut) [inline client.go]: s/clientTimeout/connectTimeout

File: kv.go [hunk 2]
```diff
@@ -181,12 +181,14 @@ func (s *tikvStore) SendKVReq(req *pb.Request, region *requestRegion) (*pb.Respo
 		if err != nil {
 			return nil, errors.Trace(err)
 		}
+		log.Warnf("addr: %s %d", region.GetAddress(), region.peer.GetId())
```
Comment (siddontang) [inline kv.go]: log.Debuf

File: kv.go [hunk 3]
```diff
@@ -181,12 +181,14 @@ func (s *tikvStore) SendKVReq(req *pb.Request, region *requestRegion) (*pb.Respo
...
 		client, err := s.getClient(region.GetAddress())
 		if err != nil {
 			return nil, errors.Trace(err)
 		}
 		req.Context = region.GetContext()
 		resp, err = client.SendKVReq(req)
+		log.Warnf("%s\n%#v", resp, err)
```
Comment (siddontang) [inline kv.go]: log.Debugf

**Comment by ngaut** [comment]
LGTM

**Pull request merged**